docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Example scenario for data warehousing and analytics in Azure
+description: Use a data warehouse in Azure to consolidate data from multiple sources and optimize data analytics.
+author: alexbuckgit
+ms.date: 08/16/2018
+---
+
+# Example scenario for data warehousing and analytics in Azure
```

Comment:
Let's update the title to highlight the business problem.  How about something like "Building an enterprise data warehouse combining multiple data sources"